### DISCUSSION TITLE
My student pack verification is not approved

### DISCUSSION THREAD

**Discussion by omerterci** [community/community#53341] [GitHub Education]
Despite the fact that I upload every document related to my school, it is persistently not approved. Even the mail I signed up for, the school mail, how much more proof do you want me to show. Transcript, student certificate I uploaded everything. There is no date on my student card, so I can't upload it. Help me, because you keep bothering me for nothing.

**Comment by samus-aran** [comment]
Hi @omerterci  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
